values.yaml

```diff
@@ -367,6 +367,11 @@ mongodb-ha:
   image:
     # StackStorm currently supports maximum MongoDB v3.4
     tag: 3.4
+  auth:
+    enabled: true
+    adminUser: "admin"
+    adminPassword: "admin"
+    key: "keykeykey"
```

Comment:
I would suggest adding a note that the default `key` should be changed, similar to the note above `secrets`.

`NB! It's highly recommended to change ALL defaults!`

Reply:
Added